compute/containers/how-to/add-a-custom-domain-to-a-container.mdx

```diff
@@ -20,7 +20,16 @@ Each container comes with an automatically generated domain on which it is avail
 
 </Message>
 
-1. Add a CNAME pointing to your container default URL from the [domains dashboard](/network/dns-cloud/how-to/manage-dns-records/).
+1. Add a CNAME pointing to your container default URL in your DNS Record (for example [Scaleway's domains](/network/dns-cloud/how-to/manage-dns-records/)).
+
+<Message type="note">
+- All domains are supported.
+- CNAME flattening* is supported.
```

Comment:
I think we should not mention a provider other than Scaleway!

Reply:
Cloudflare is however a major player in the world of DNS and the web in general. Everyone knows Cloudflare, everyone browses sites protected, cached, etc. by Cloudflare. I think it would be a mistake not to talk about them here.

Moreover, they seem to be the only ones to support CNAME flattening.

Reply:
@bene2k1 what do you think about it ?

Reply:
I am okay with mentioning them here if they are the only ones providing this feature

Reply:
Pris en compte